src/comet_llm/config.py

```diff
@@ -39,6 +39,9 @@ def _extend_comet_ml_config() -> None:
         "comet.logging.console": {"type": str, "default": "INFO"},
         "comet.raise_exceptions_on_error": {"type": int, "default": 0},
         "comet.internal.check_tls_certificate": {"type": bool, "default": True},
+        "comet.offline": {"type": bool, "default": False},
+        "comet.offline_folder_path": {"type": str, "default": ".cometllm-runs"},
```

Comment:
```suggestion
        "comet.offline_directory": {"type": str, "default": ".cometllm-runs"},
```
Can we use the same config name as `comet_ml` for consistency?